app/core/tests/test_models_integration.py

```diff
@@ -90,3 +90,25 @@ def test_save_course_detail_highlight_success(self):
 
         self.assertEqual(retrievedObj.field_name, field)
         self.assertEqual(retrievedObj.display_name, name)
+
+    def test_save_course_information_mapping_failure(self):
```

Comment:
I'm not really seeing here how we are testing the mapping failure with this test -- shouldn't we be looking for the raised ValidationError?

Reply:
Yes, and it has been fixed now. Thanks for spotting that.